src/app/auth/components/signup/signup.component.html

```diff
@@ -112,7 +117,12 @@ <h1 class="page-title" *ngIf="isForShareInvite">
       [icon]="'/auth/sign-up'"
       [orientation]="'right'"
       [style]="{ justifyContent: 'center' }"
-      [disabled]="waiting || signupForm.invalid || !agreedTerms"
+      [disabled]="
+        waiting ||
+        signupForm.invalid ||
+        !agreedTerms ||
+        passwordStrengthMessage !== 'Strong'
```

Comment:
I'm not sure if we want to block people from signing up just based on password strength. The intention is to show an indicator so that people can see their password strength and make their own decision on if they want to pick a stronger password.